src/components/FileUploader/FileUploader.js

```diff
@@ -13,11 +13,9 @@ export class FileUploaderButton extends Component {
     disableLabelChanges: PropTypes.bool,
     id: PropTypes.string,
     labelText: PropTypes.string,
-    listFiles: PropTypes.bool,
     multiple: PropTypes.bool,
     name: PropTypes.string,
     onChange: PropTypes.func,
-    onClick: PropTypes.func,
```

Comment:
Why did you remove the prop type for `onClick` here…?

Reply:
correct me if I'm wrong but I don't think this prop is being used at all since a click handler is already hard coded [here](https://github.com/IBM/carbon-components-react/pull/1105/files/8fb14765a5ce8307bf2b7edfab49ffdec075da9d#diff-b82c23ac01d092b1d134837688198215L106)

Reply:
@emyarod As far as I read the original code correctly, `onClick` of `<FileUploaderButton>` goes [here](https://github.com/IBM/carbon-components-react/blob/3677e8a/src/components/FileUploader/FileUploader.js#L94), but don’t hesitate to let me know if you think otherwise. Thanks!

Reply:
My mistake @asudoh you're right, I overlooked the spread props again. I've made the correction